fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg

```diff
@@ -143,23 +145,23 @@ point.22 = 2,79.979874,0.0,21.776155,100,0,0.0001,0,1,0.0001,0,0,0,4,0,0,1 ; Bod
+Tank.3 = Name:LeftMid#Title:FUEL LEFT MID#Capacity:9632#UnusableCapacity:0#Position:7.1,-45.9,5.9#Priority:3#InputOnlyLines:LeftMidFwdXferValveToLeftMidTank,LeftMidAftXferValve2ToLeftMidTank#OutputOnlyLines:LeftMidTankToLeftMidTankPumpFwd,LeftMidTankToLeftMidTankPumpAft
+Tank.4 = Name:LeftInner#Title:FUEL LEFT INNER#Capacity:12189.4#UnusableCapacity:0#Position:16.5,-24.7,3.2#Priority:4#InputOnlyLines:LeftInnerFwdXferValveToLeftInnerTank,LeftInnerAftXferValve2ToLeftInnerTank#OutputOnlyLines:LeftInnerTankToLeftInnerTankPumpFwd,LeftInnerTankToLeftInnerTankPumpAft
+Tank.5 = Name:Feed2#Title:FUEL FEED TWO#Capacity:7753.2.#UnusableCapacity:0#Position:27.3,-18.4,1.0#Priority:5#InputOnlyLines:FeedTank2FwdXferValve1_1ToFeedTank2,FeedTank2FwdXferValve1_2ToFeedTank2,FeedTank2FwdXferValve2_1ToFeedTank2,FeedTank2FwdXferValve2_2ToFeedTank2,FeedTank2AftXferValve2ToFeedTank2#OutputOnlyLines:FeedTank2ToFeed2TankPump1,FeedTank2ToFeed2TankPump2
+Tank.6 = Name:Feed3#Title:FUEL FEED THREE#Capacity:7753.2#UnusableCapacity:0#Position:27.3,18.4,1.0#Priority:5#InputOnlyLines:FeedTank3FwdXferValve1_1ToFeedTank3,FeedTank3FwdXferValve1_2ToFeedTank3,FeedTank3FwdXferValve2_1ToFeedTank3,FeedTank3FwdXferValve2_2ToFeedTank3,FeedTank3AftXferValve2ToFeedTank3#OutputOnlyLines:FeedTank3ToFeed3TankPump1,FeedTank3ToFeed3TankPump2
+Tank.7 = Name:RightInner#Title:FUEL RIGHT INNER#Capacity:12189.4#UnusableCapacity:0#Position:16.5,24.7,3.2#Priority:4#InputOnlyLines:RightInnerFwdXferValveToRightInnerTank,RightInnerAftXferValve2ToRightInnerTank#OutputOnlyLines:RightInnerTankToRightInnerTankPumpFwd,RightInnerTankToRightInnerTankPumpAft
+Tank.8 = Name:RightMid#Title:FUEL RIGHT MID#Capacity:9632#UnusableCapacity:0#Position:7.1,45.9,5.9#Priority:3#InputOnlylines:RightMidFwdXferValveToRightMidTank,RightMidAftXferValve2ToRightMidTank#OutputOnlyLines:RightMidTankToRightMidTankPumpFwd,RightMidTankToRightMidTankPumpAft
+Tank.9 = Name:Feed4#Title:FUEL FEED FOUR#Capacity:7299.6#UnusableCapacity:0#Position:-7.45,71,7.3#Priority:5#InputOnlyLines:FeedTank4FwdXferValve1ToFeedTank4,FeedTank4FwdXferValve2ToFeedTank4,FeedTank4AftXferValve2ToFeedTank4#OutputOnlyLines:FeedTank4ToFeed4TankPump1,FeedTank4ToFeed4TankPump2
+Tank.10 = Name:RightOuter#Title:FUEL RIGHT OUTER#Capacity:2731.5#UnusableCapacity:0#Position:-25.0,100,8.5#Priority:2#InputOnlyLines:RightOuterFwdXferValveToRightOuterTank,RightOuterAftXferValve2ToRightOuterTank#OutputOnlyLines:RightOuterTankToRightOuterTankPump,RightOuterTankToRightOuterEmerXferValve
+Tank.11 = Name:Trim#Title:FUEL TRIM#Capacity:6260.3#UnusableCapacity:0#Position:-87.14,0,12.1#Priority:1#InputOnlyLines:TrimTankInletValveToTrimTank#OutputOnlyLines:TrimTankToTrimTankPumpLeft,TrimTankToTrimTankPumpRight
```

Comment:
The inputline doesnt exist anymore and probably should refer to the two new lines